src/assistant/assistant-helper.js

```diff
@@ -59,15 +59,15 @@ const startServer = () => {
 const getBuilder = async (url, { useExisting = false, outputPath }) => {
   console.log(chalk.magenta('Import assistant is analyzing the page...'));
   const auth = {
-    authCode: process.env.IMS_AUTH_CODE,
-    clientSecret: process.env.IMS_CLIENT_SECRET,
+    apiKey: process.env.AEM_IMPORT_API_KEY,
+    environment: 'prod', // or 'stage'
```

Comment:
Can you make this configurable via the CLI?

The import command already includes a `stage` option. This should be added to the assistant commands as well and and used to set the environment accordingly. Default should be prod.

https://github.com/adobe/aem-import-helper/blob/main/src/cmd/import.js#L41